content/docs/0.8.1/deploy/install/_index.md

```diff
@@ -34,7 +34,7 @@ Each node in the Kubernetes cluster where Longhorn is installed must fulfill the
 ### OS/Distro Specific Configuration
 
 - **Google Kubernetes Engine (GKE)** requires some additional setup for Longhorn to function properly. If you're a GKE user, refer to [this section](../../advanced-resources/os-distro-specific/csi-on-gke) for details.
-- **K3s clusters** require some extra setup. Refer to [this section](../../advanced-resources/os-distro-specific/csi-on-k3s)
+- **K3s clusters** require some extra setup. Refer to [this section.](../../advanced-resources/os-distro-specific/csi-on-k3s)
```

Comment:
We might need to update k3s information. Recently there are a few improvements and we need to check if it still needs custom setups.

Reply:
I opened this issue https://github.com/longhorn/longhorn/issues/1352 to track the updates for that page